refarch-gateway/src/main/java/de.muenchen.oss.refarch.gateway/configuration/SecurityConfiguration.java

```diff
@@ -78,6 +78,8 @@ public SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http)
                             .pathMatchers(LOGOUT_SUCCESS_URL).permitAll()
                             .pathMatchers("/api/*/info",
                                     "/actuator/health",
+                                    "/actuator/health/liveness",
+                                    "/actuator/health/readiness",
```

Comment:
Would be `/actuator/*` be easier to manage? Or is this to loosely defined?

Reply:
Thought as restrictive as possible makes sense here. I would leave it like this for the moment as also every other projects defines the matchers without wildcards.
Also I already created #36 to hide the whole actuator endpoint from the outside.